querybook/server/lib/elasticsearch/suggest_table.py

```diff
@@ -1,17 +1,18 @@
 def construct_suggest_table_query(
-    prefix: str,
+    keyword: str,
     limit: int,
     metastore_id: int,
 ):
     return {
-        "suggest": {
-            "suggest": {
-                "text": prefix,
-                "completion": {
-                    "field": "completion_name",
```

Comment:
since completion name is getting removed, can you add some comment in the elasticsearch definition on that field so we know to delete it for future? Also I guess we can stop indexing any data to that field